### ISSUE TITLE
Question submissions being re-scored as null

### ISSUE THREAD

**Issue by jakebailey** [PrairieLearn/PrairieLearn#578]
Question submissions seem to be getting re-scored for a few students. Here is an example event log for a single question

```
{"submitted_answer":{"selection":"a0"}}

{"points":3,"max_points":3,"score_perc":100}

{"score":1,"correct":true,"feedback":null,"true_answer":{"answer":"True","checks":[true,false],"explanation":"A full adder is designed to take three input bits, the two from the two numbers being added and a third for the carry in."},"submitted_answer":{"selection":"a0"}}

{"points":null,"max_points":3,"score_perc":null}
```

This results in points that look like:
![image](https://cloud.githubusercontent.com/assets/5341706/22723386/7a80fdc6-ed83-11e6-866f-b7918815797a.png)

And the nulls are treated as zeros. Also note that the number of attempts is >1, except if I open one of the questions:
![image](https://cloud.githubusercontent.com/assets/5341706/22723398/959a158e-ed83-11e6-819b-5ff265f9ab5a.png)

There's just a single saved answer.

I can't reproduce this, and I don't know where to start looking. If it's not fixed, I'll have to open all 200 exams and check the submissions, which I don't want to do. :(

**Comment by mwest1066** [comment]
Fixed in ae9c6497090b457d725dc18dd07de96468a83fe2

**Comment by mwest1066** [comment]
We apparently fixed this at some point.
